### ISSUE TITLE
Return Trivy DB update timestamp in metadata

### ISSUE THREAD

**Issue by danielpacak** [aquasecurity/harbor-scanner-trivy#48]
For Clair Harbor was able to display the upstream vulnerabilities DB update timestamp. If possible we should support the same, i.e. add the `harbor.scanner-adapter/vulnerability-database-updated-at` property to the metadata response.

```
curl -k https://harbor-scanner-trivy:8443/api/v1/metadata
```

```js
{
  "scanner": {
    "name": "Trivy",
    "vendor": "Aqua Security",
    "version": "0.1.7",
  },
  "capabilities": [
    {
      "consumes_mime_types": [
        "application/vnd.oci.image.manifest.v1+json",
        "application/vnd.docker.distribution.manifest.v2+json"
      ],
      "produces_mime_types": [
        "application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0",
        "application/vnd.scanner.adapter.vuln.report.raw"
      ]
    }
  ],
  "properties": {
    "harbor.scanner-adapter/scanner-type": "os-package-vulnerability",
    "harbor.scanner-adapter/vulnerability-database-updated-at": "2019-08-13T08:16:33.345Z",
    "harbor.scanner-adapter/vulnerability-database-next-update-at": "2019-08-14T08:16:33.345Z"
  }
}
```

**Comment by danielpacak** [comment]
@knqyf263

**Comment by danielpacak** [comment]
Resolved by #87
